utils_test.go

```diff
@@ -37,3 +40,9 @@ func mustNoError(t *testing.T, err error) {
 		t.FailNow()
 	}
 }
+
+func mustError(t *testing.T, err error) {
```

Comment:
Looks like an essential feature for testify. I've skipped that argument, due to your time pressure. 😁

Reply:
:+1: